doc/06_contributing.md

```diff
@@ -12,20 +12,20 @@ into the idea.
 
 Some rules to follow:
 
-* Use Github! Sign up for an account, fork our repository and send a
+* Use GitHub! Sign up for an account, fork our repository and send a
   pull request after you're satisfied with your work. We won't accept
-  patches send by mail or - even worse - as pastebin links.
+  patches sent by email or - even worse - as pastebin links.
```

Comment:
email? shouldn't it be e-mail or E-Mail or something?

Reply:
[English seems to use email more often than e-mail](https://grammarist.com/style/e-mail-email/), so I opted for that form.